src/platforms/android/common/proguard.mdx

```diff
@@ -76,5 +79,23 @@ sentry {
 }
 ```
 
-And that's it! Now when you build your app, the plugin will upload the ProGuard/R8 mappings,
-source bundle, and native symbols, as you configured them to Sentry.
+Now when you build your app, the plugin will upload the ProGuard/R8 mappings, source bundle, and native symbols as you configured them to Sentry.
+
+### Multi-Dex Support
```

Comment:
I don't think this should be under this section.

the configuration is similar to the proguard way, but it's not related to it.

I also think we don't have a suitable section for it, it's more a general advanced config, or caveats or something like that

Reply:
maybe we create a section under platforms/android/configuration/ ?

Reply:
That was my attempt :) But, yes, I'll make it its own page under Config. Thanks for double checking, @marandaneto